jobs/build/custom/Jenkinsfile

```
@@ -230,8 +232,9 @@ node {
 
             stage("build images") {
                 if (!any_images_to_build) { return }
+                buildlib.registry_quay_qe_login()
                 command = doozerOpts
-                command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults"
+                command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults --registry-config=./qe_quay_config"
```

`custom` could build either ocp3 or ocp4 images. you'll need to authenticate to the right registry depending, and then pass the right config here.

also if you follow my suggestion in the doozer PR this will become:
```suggestion
                command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults --registry-config-dir=./qe_quay_config"
```